experimental/GModule/src/GModule.jl

```diff
@@ -949,11 +1012,17 @@ end
 function _regular_gmodule(G::Group, M)
   ge = collect(G)
   ZG = gmodule(G, [hom(M, M, [M[findfirst(isequal(ge[i]*g), ge)] for i=1:length(ge)]) for g = gens(G)])
+  set_attribute!(ZG, :is_regular => ge)
   return ZG, C->(x -> sum(x[i]*action(C, ge[i]) for i=1:length(ge))),
     MapFromFunc(G, ZZ, x->ZZ(findfirst(isequal(x), ge)),
              y->ge[Int(y)])
 end
 
+function is_regular_gmodule(G::GModule)
+  return get_attribute(G, :is_regular) !== nothing
```

Comment:
```suggestion
  return has_attribute(G, :is_regular)
```
Why not just this?

Reply:
Actually, shouldn't it in fact be *this* ? I.e. we don't just want to check whether the `is_regular` attribute is set, but also whether it is set to `true`, right?
```suggestion
  return get_attribute(G, :is_regular, false)
```